src/library/vm/vm_io.cpp

```diff
@@ -437,7 +447,8 @@ static vm_obj io_get_env(vm_obj const & k, vm_obj const &) {
 }
 
 static vm_obj io_get_cwd(vm_obj const &) {
-    auto cwd = getcwd(nullptr, 0);
+    char buffer[PATH_MAX];
+    auto cwd = getcwd(buffer, sizeof(buffer));
     if (cwd) {
```

Comment:
I saw you included extra files for Linux, what about OSX?

Reply:
I'm watching the build bots. I'll make sure they are all green.

Reply:
Thanks!

Reply:
btw this also fixes a memory leak here. The buffer returned by getcwd wasn't being free().